packages/remix-dev/cli/create.ts

```diff
@@ -380,6 +411,40 @@ function getGithubUrl(info: Omit<RepoInfo, "url">) {
   return url;
 }
 
+function isGithubReleaseAssetUrl(url: string) {
+  return (
+    url.startsWith("https://github.com") && url.includes("/releases/download/")
+  );
+}
+interface ReleaseAssetInfo {
+  browserUrl: string;
+  owner: string;
+  name: string;
+  asset: string;
+  tag: string;
+}
+function getGithubReleaseAssetInfo(browserUrl: string): ReleaseAssetInfo {
+  // for example, https://github.com/deptagency/dash/releases/download/v0.0.1/stack.tar.gz
```

Comment:
do you mind using an anonymous example here? (this could be perceived as promoting stuff)

```suggestion
  // for example, https://github.com/owner/repository/releases/download/v0.0.1/stack.tar.gz
```

Reply:
no problem, that's just the url I was testing with.